src/LndClient/Data/Channel.hs

```diff
@@ -37,10 +37,10 @@ instance FromGrpc Channel GRPC.Channel where
     Channel
       <$> fromGrpc (GRPC.channelRemotePubkey x)
       <*> channelPointParser (GRPC.channelChannelPoint x)
-      <*> fromGrpc (GRPC.channelCapacity x)
-      <*> fromGrpc (GRPC.channelLocalBalance x)
-      <*> fromGrpc (GRPC.channelRemoteBalance x)
-      <*> fromGrpc (GRPC.channelCommitFee x)
+      <*> (toMSat <$> fromGrpc (GRPC.channelCapacity x))
```

Comment:
Why not to implement `fromGrpc` directly for `MSat`?

Reply:
Or `channelCapacity` have `Sat` only value?

Reply:
yes, channelCapacity returns Int64 (Sat) only value
so we either should specify field channelCapacity as Sat (in this case fromGrpc Sat Int64 can be implemented) or leave is MSat as is and convert explicitly

Reply:
For fields which don't have mSat value, this is ok